docs/pwa-service-worker-cache-control.md

```
@@ -0,0 +1,279 @@
+```bash
+sudo a2enmod headers
+sudo systemctl restart apache2
+```
+
+---
+
+### Netlify
+
+Create or update a `_headers` file in your project root (or public directory):
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)